cms/templates/export.html

```diff
@@ -70,10 +72,10 @@ <h2 class="title">${_("About Exporting Libraries")}</h2>
           <h2 class="title">${_("About Exporting Courses")}</h2>
           <div class="copy">
               ## Translators: ".tar.gz" is a file extension, and should not be translated
-              <p>${_("You can export courses and edit them outside of {studio_name}. The exported file is a .tar.gz file (that is, a .tar file compressed with GNU Zip) that contains the course structure and content. You can also re-import courses that you've exported.").format(
+              <p>${Text(_("You can export courses and edit them outside of {studio_name}. The exported file is a .tar.gz file (that is, a .tar file compressed with GNU Zip) that contains the course structure and content. You can also re-import courses that you've exported.")).format(
```

Comment:
Generally we leave out the `Text()` calls unless they are needed, and they aren't needed unless you are interpolating an `HTML()` call into the string.